package/gluon-mesh-vpn-fastd/luasrc/usr/lib/lua/gluon/mesh-vpn/method/fastd.lua

```diff
@@ -0,0 +1,39 @@
+local uci = require("simple-uci").cursor()
+
+local site = require 'gluon.site'
+local util = require "gluon.util"
+
+local M = {}
+
+function M.public_key()
+	return util.trim(util.exec("/etc/init.d/fastd show_key mesh_vpn"))
+end
+
+function M.enable(val)
+	uci:set("fastd", "mesh_vpn", "enabled", val)
+	uci:save("fastd")
+end
+
+function M.active()
```

Comment:
Same as in https://github.com/freifunk-gluon/gluon/pull/2102/files#r626032365
```suggestion
function M.available()
```